cli: show the issue in a better way (#536)
radicle-dev/heartwood: radicle-cli/examples/workflow/3-issues.md

╰─────────────────────────────────────────────────────────────────────────────────────────────────────╯
```

> In addition, you can see that when you run `rad issue show` the issue add you to `Assignees`.

Reviewer: ```suggestion
In addition, you can see that when you run `rad issue show` you are listed under the `Assignees`.
```

╰─────────────────────────────────────────────────────────────────────────────────────────────────────╯
```

In addition, you can see that when you run `rad issue show` the issue add you to `Assignees`.

```

> $ rad issue show b05e945

Reviewer: It's weird that the output form the `rad issue list --assigned` command above doesn't show `You` but this one does. I think this is what @cloudhead mentioned that we need to standardise how this is done otherwise it will sometimes show `You` and sometimes show your `NID`, and maybe even sometimes show your `DID`.

Probably worth discussing how this should be done before making everything drift apart visually :)

Author: for now I restore the `did`
